simulation/traffic_simulator/config/scenario_simulator_v2.rviz

```diff
@@ -3056,10 +3056,10 @@ Visualization Manager:
                 Value: /perception/occupancy_grid_map/map_updates
               Use Timestamp: false
               Value: false
-          Enabled: true
+          Enabled: false
           Name: Simple Sensor Simulator
-      Enabled: true
-      Name: Simulation
+      Enabled: false
+      Name: scenario simulator v2 information
```

Comment:
```suggestion
      Name: scenario_simulator_v2
```

Reply:
@yamacir-kit 
Changed name from scenario simulator v2 information to Simulation